documentation/source/install/install_usecases.rst

```diff
@@ -201,3 +201,157 @@ The above command can be invoked using the *watch* command by providing a desire
 
    watch -n 1 maya vsm-stats pvc-016e9a68-71c1-11e7-9fea-000c298ff5fc
 
+******************
+Usecases - Jupyter
+******************
+ 
+Running Jupyter on OpenEBS
+===========================
+
+This section provides detailed instructions on how to run a *jupyter-mysql* application pod on OpenEBS storage in a Kubernetes cluster and uses a *mysql-client* container to generate load in order to illustrate input/output traffic on the storage.
```

Comment:
This section provides detailed instructions on how to run a *jupyter* pod on OpenEBS storage in a Kubernetes cluster and uses a *jupyter ui editor* to generate load in order to illustrate input/output traffic on the storage.

Reply:
Done!